projects/components/src/breadcrumbs/breadcrumbs.component.test.ts

```diff
@@ -13,7 +15,8 @@ describe('BreadcrumbsComponent', () => {
   const createHost = createHostFactory({
     declareComponent: false,
     component: BreadcrumbsComponent,
-    imports: [BreadcrumbsModule, HttpClientTestingModule, IconLibraryTestingModule],
+    imports: [BreadcrumbsModule, HttpClientTestingModule, IconLibraryTestingModule, RouterTestingModule],
+    declarations: [MockDirective(TrackDirective)],
```

Comment:
So here's an example. The mocked track component should be sufficient, we shouldn't need the router testing module too (and in general, this test needs to be converted be shallow)

Reply:
So, even the MockDirective still tries to inject Router since it is in the constructor.

Reply:
really? that seems off...

Reply:
Yep, they try to inject the services. I have seen it at few other places.